utility/test/account_test.go

```diff
@@ -168,76 +167,82 @@ func TestUtilityContext_SetAccountWithAmountString(t *testing.T) {
+	ctx.SetPoolAmount(pool.GetAddress(), beforeAmountBig)
 	subAmountBig := big.NewInt(100)
 	subAmount := types.BigIntToString(subAmountBig)
-	require.NoError(t, ctx.SubPoolAmount(pool.Address, subAmount), "sub pool amount")
-	amount, err := ctx.GetPoolAmount(pool.Address)
+	require.NoError(t, ctx.SubPoolAmount(pool.GetAddress(), subAmount), "sub pool amount")
+	amount, err := ctx.GetPoolAmount(pool.GetAddress())
 	require.NoError(t, err)
 	require.True(t, beforeAmountBig.Cmp(amount) != 0, fmt.Sprintf("no amount change in pool"))
 	expected := beforeAmountBig.Sub(beforeAmountBig, subAmountBig)
 	require.True(t, expected.Cmp(amount) == 0, fmt.Sprintf("unexpected pool amount; expected %v got %v", expected, amount))
-	tests.CleanupTest(ctx)
+	test_artifacts.CleanupTest(ctx)
 }
 
 func TestUtilityContext_SubtractAccountAmount(t *testing.T) {
 	ctx := NewTestingUtilityContext(t, 0)
 	acc := GetAllTestingAccounts(t, ctx)[0]
 
-	beforeAmount := acc.Amount
+	beforeAmount := acc.GetAmount()
 	beforeAmountBig, err := types.StringToBigInt(beforeAmount)
 	require.NoError(t, err)
 
 	subAmountBig := big.NewInt(100)
-	addrBz, er := hex.DecodeString(acc.Address)
+	addrBz, er := hex.DecodeString(acc.GetAddress())
 	require.NoError(t, er)
 	require.NoError(t, ctx.SubtractAccountAmount(addrBz, subAmountBig), "sub account amount")
 	amount, err := ctx.GetAccountAmount(addrBz)
 	require.NoError(t, err)
 	require.True(t, beforeAmountBig.Cmp(amount) != 0, fmt.Sprintf("no amount change in pool"))
 	expected := beforeAmountBig.Sub(beforeAmountBig, subAmountBig)
 	require.True(t, expected.Cmp(amount) == 0, fmt.Sprintf("unexpected acc amount; expected %v got %v", expected, amount))
-	tests.CleanupTest(ctx)
+	test_artifacts.CleanupTest(ctx)
 }
 
-func GetAllTestingAccounts(t *testing.T, ctx utility.UtilityContext) []*genesis.Account {
-	accs, err := (ctx.Context.PersistenceRWContext).(persistence.PostgresContext).GetAllAccounts(0)
+func GetAllTestingAccounts(t *testing.T, ctx utility.UtilityContext) (acs []modules.Account) {
+	accs, err := (ctx.Context.PersistenceRWContext).GetAllAccounts(0)
+	require.NoError(t, err)
 	sort.Slice(accs, func(i, j int) bool {
-		return accs[i].Address < accs[j].Address
+		return accs[i].GetAddress() < accs[j].GetAddress()
 	})
-	require.NoError(t, err)
-	return accs
+	for _, acc := range accs {
+		acs = append(acs, acc)
+	}
+	return
 }
 
-func GetAllTestingPools(t *testing.T, ctx utility.UtilityContext) []*genesis.Account {
-	accs, err := (ctx.Context.PersistenceRWContext).(persistence.PostgresContext).GetAllPools(0)
+func GetAllTestingPools(t *testing.T, ctx utility.UtilityContext) (acs []modules.Account) {
+	accs, err := (ctx.Context.PersistenceRWContext).GetAllPools(0)
+	require.NoError(t, err)
 	sort.Slice(accs, func(i, j int) bool {
-		return accs[i].Address < accs[j].Address
+		return accs[i].GetAddress() < accs[j].GetAddress()
 	})
-	require.NoError(t, err)
-	return accs
+	for _, acc := range accs {
```

Comment:
I made this change and all the tests still passed. Not pushing yet because I wanted to double check I'm not missing something obvious:

<img width="880" alt="Screen Shot 2022-09-12 at 7 20 33 PM" src="https://user-images.githubusercontent.com/1892194/189774911-2d540cc4-ced1-4d2a-853a-9817bd362637.png">